sklearn/neural_network/tests/test_mlp.py

```diff
@@ -418,9 +417,9 @@ def test_partial_fit_regression():
             mlp.partial_fit(X, y)
 
         pred2 = mlp.predict(X)
-        assert_almost_equal(pred1, pred2, decimal=2)
+        assert_almost_equal(pred1, pred2)
```

Comment:
Removed `decimal=2` to use the default 7 as the predictions should be/are equal to more precision.